scripts/install-linters.sh

```diff
@@ -1,29 +1,43 @@
-sudo add-apt-repository --yes ppa:kalakris/cmake
+set -x
+set -e
+
+root="$(git rev-parse --show-toplevel)"
+${root}/scripts/clean-linters.sh
```

Comment:
Why is this required? Were the listed linters sometimes installed in the environment?